mkdocs.yml

```diff
@@ -45,7 +45,9 @@ nav:
   - Introduction: introduction.md
   - Installation: installation.md
   - Getting Started: getting_started.md
-  - User Guide: user_guide.md
+  - User Guide:
+      - Testing: user_guide/testing.md
+      - Best Practices: content_best_practices.md
```

Comment:
This should get those sections onto the main navigation. I don't think we still need user_guide.md in this case?